packages/lib-user/src/components/UserHome/components/Dashboard/components/StatsTabs/StatsTabsContainer.js

```diff
@@ -20,25 +21,19 @@ const formatStatsPreview = (thisWeekData, allTimeData) => {
   }
 }
 
+const DEFAULT_DATE_RANGE = getDefaultDateRange() // last 7 days
+
 export default function StatsTabsContainer({ user }) {
-  const today = new Date()
-  const todayUTC = getStatsDateString(today)
-  
-  const thisWeekStart = new Date()
-  thisWeekStart.setUTCDate(today.getUTCDate() - 6)
-  const thisWeekQuery = getDateInterval({
-    endDate: todayUTC,
-    startDate: getStatsDateString(thisWeekStart)
-  })
+  const thisWeekQuery = getDateInterval(DEFAULT_DATE_RANGE)
   thisWeekQuery.project_contributions = true
   const { data: thisWeekData } = useStats({
     sourceId: user?.id,
     query: thisWeekQuery
   })
 
   const allTimeQuery = getDateInterval({
-    endDate: todayUTC,
-    startDate: getStatsDateString(user?.created_at)
+    endDate: getStatsDateString(new Date()), // Today's UTC date string like 24-09-23.
```

Comment:
Pedantic, but not sure if the `20` in `2024` should be included
```suggestion
    endDate: getStatsDateString(new Date()), // Today's UTC date string like 2024-09-23.
```